app/docs/components/datepicker/datepicker.mdx

```diff
@@ -0,0 +1,97 @@
+import { CodePreview } from '~/app/components/code-preview';
```

Comment:
It would be important to add more examples regarding the most important API props (like `defaultDate`)